projects/1.20-fabric/assets/would/would/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+{
+  "itemGroup.would.would": "Would",
+
+  "block.would.willow_log": "柳木原木",
+  "block.would.baobab_log": "猴面包原木",
+  "block.would.ebony_log": "黑檀原木",
+  "block.would.ebony_heartwood_log": "黑檀心原木",
+  "block.would.fir_log": "冷杉原木",
+  "block.would.pine_log": "松木原木",
+  "block.would.cedar_log": "雪松原木",
+  "block.would.mahogany_log": "桃花心木原木",
+  "block.would.azalea_log": "杜鹃原木",
+  "block.would.palm_log": "棕榈原木",
+  "block.would.maple_log": "枫木原木",
+  "block.would.maple_log_sappy": "多汁枫木原木",
+  "block.would.aspen_log": "白杨原木",
+  "block.would.aspen_log_gazing": "启眸白杨原木",
+  "block.would.walnut_log": "胡桃木原木",
+
+  "block.would.willow_wood": "柳木",
+  "block.would.baobab_wood": "猴面包木",
+  "block.would.ebony_wood": "黑檀木",
+  "block.would.ebony_heartwood": "黑檀心木",
+  "block.would.fir_wood": "冷杉木",
+  "block.would.pine_wood": "松木",
+  "block.would.cedar_wood": "雪松木",
+  "block.would.mahogany_wood": "桃花心木",
+  "block.would.azalea_wood": "杜鹃木",
+  "block.would.palm_wood": "棕榈木",
+  "block.would.maple_wood": "枫木",
+  "block.would.aspen_wood": "白杨木",
+  "block.would.aspen_wood_gazing": "启眸白杨木",
+  "block.would.walnut_wood": "胡桃木",
+
+  "block.would.stripped_willow_log": "去皮柳木原木",
+  "block.would.stripped_baobab_log": "去皮猴面包原木",
+  "block.would.stripped_ebony_log": "去皮黑檀原木",
+  "block.would.stripped_fir_log": "去皮冷杉原木",
+  "block.would.stripped_pine_log": "去皮松木原木",
+  "block.would.stripped_cedar_log": "去皮雪松原木",
+  "block.would.stripped_mahogany_log": "去皮桃花心木原木",
+  "block.would.stripped_azalea_log": "去皮杜鹃原木",
+  "block.would.stripped_palm_log": "去皮棕榈原木",
+  "block.would.stripped_maple_log": "去皮枫木原木",
+  "block.would.stripped_aspen_log": "去皮白杨原木",
+  "block.would.stripped_aspen_log_gazing": "去皮启眸白杨原木",
+  "block.would.stripped_walnut_log": "去皮胡桃木原木",
+
+  "block.would.stripped_willow_wood": "去皮柳木",
+  "block.would.stripped_baobab_wood": "去皮猴面包木",
+  "block.would.stripped_ebony_wood": "去皮黑檀木",
+  "block.would.stripped_fir_wood": "去皮冷杉木",
+  "block.would.stripped_pine_wood": "去皮松木",
+  "block.would.stripped_cedar_wood": "去皮雪松木",
+  "block.would.stripped_mahogany_wood": "去皮桃花心木",
+  "block.would.stripped_azalea_wood": "去皮杜鹃木",
+  "block.would.stripped_palm_wood": "去皮棕榈木",
+  "block.would.stripped_maple_wood": "去皮枫木",
+  "block.would.stripped_aspen_wood": "去皮白杨木",
+  "block.would.stripped_aspen_wood_gazing": "去皮启眸白杨木",
+  "block.would.stripped_walnut_wood": "去皮胡桃木",
+
+  "block.would.willow_leaves": "柳树树叶",
+  "block.would.baobab_leaves": "猴面包树树叶",
+  "block.would.ebony_leaves": "黑檀树树叶",

```

```suggestion
  "block.would.ebony_leaves": "黑檀树叶",
```
同原版“白桦树叶”“白桦树苗”，树苗部分统一修改